source/shared/cpp/ObjectModel/FactSet.cpp

```diff
@@ -70,4 +73,10 @@ std::shared_ptr<BaseCardElement> FactSetParser::DeserializeFromString(
     const std::string& jsonString)
 {
     return FactSetParser::Deserialize(elementParserRegistration, actionParserRegistration, ParseUtil::GetJsonValueFromString(jsonString));
-}
\ No newline at end of file
+}
+
+void FactSet::PopulateKnownPropertiesSet(void) 
+{
+    m_knownProperties.insert(AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Title));
+    m_knownProperties.insert(AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Value));
```

Comment:
This should be just Facts. Title and Value are properties of the individual Facts, not the FactSet.